### DISCUSSION TITLE
[error] Cannot resolve vue-i18n on pnpm! please install it on 'node_modules'

### DISCUSSION THREAD

**Discussion by baixiaoyu2997** [nuxt-modules/i18n#2519] [General]
I had this problem on vercel， the problem was solved by removing `extends: ['./extends/i18n']` in `/extends/base/nuxt.config.ts` in `base`,
and nuxt.config in `root`,alse have extends config,  `extends: [extends/base]`

Sorry I don't have time to do a minimal reproduction

**Comment by angelhdzmultimedia** [comment]
Same here.

**Edit:** Solved. Created `.npmrc` file with `shamefully-hoist=true`. Then ran `pnpm install`. Now working. 
My issue is specific, when I run `pnpx nuxi@latest init .`, it fails to create the `.npmrc`. I have an open issue about this.

Hope this help anyone.
